zachsinterview/src/main/java/com/imprev/zachsinterview/Main.java

```diff
@@ -0,0 +1,37 @@
+package com.imprev.zachsinterview;
+
+public class Main {
+
+    /**
+     * Demonstrates how to connect to thirdPartyAPI and tests how the numerical key store may be used.
+     */
+    public static void main(String[] argv) throws NumberStoreException {
```

Comment:
general comment: there do not appear to be any test cases, aside from this driver code